dask/dataframe/core.py

```diff
@@ -7349,3 +7409,45 @@ def series_map(base_series, map_series):
     divisions = list(base_series.divisions)
 
     return new_dd_object(graph, final_prefix, meta, divisions)
+
+
+def _monotonic_increasing_chunk(x):
+    return pd.DataFrame(
+        data=[[x.is_monotonic_increasing, x.iloc[0], x.iloc[-1]]],
+        columns=["monotonic", "first", "last"],
+    )
+
+
+def _index_monotonic_increasing_chunk(x):
+    return pd.DataFrame(
+        data=[[x.is_monotonic_increasing, x[0], x[-1]]],
+        columns=["monotonic", "first", "last"],
+    )
+
+
+def _monotonic_increasing_aggregate(concatenated):
+    bounds_are_monotonic = pd.Series(
+        concatenated[["first", "last"]].to_numpy().ravel()
+    ).is_monotonic_increasing
+    return concatenated["monotonic"].all() and bounds_are_monotonic
+
+
+def _monotonic_decreasing_chunk(x):
+    return pd.DataFrame(
+        data=[[x.is_monotonic_decreasing, x.iloc[0], x.iloc[-1]]],
+        columns=["monotonic", "first", "last"],
+    )
+
+
+def _index_monotonic_decreasing_chunk(x):
+    return pd.DataFrame(
```

Comment:
It just occurred to me that for index there should be a fast track if `.divisions` exists since then we can just check whether the `divisions` are monotonically increasing.

Reply:
eh I guess that doesn't add much though since it just simplies the `bounds_are_monotonic` logic below.

Reply:
I opted out for not including, since I guess I would have to add logic to handle it since I merge the methods for Index and Series